src/validation/fields/email.validation.ts

```diff
@@ -1,17 +1,12 @@
 import { body } from "express-validator";
-import { ErrorMessages } from "../error.messages";
-import { VALID_EMAIL_FORMAT } from "../regex/regex.validation";
+import { validateEmail } from "../custom.validation";
 
 export const email_validations = [
   body("email").trim()
-    .not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.EMAIL)
-    .isLength({ max: 256 }).withMessage(ErrorMessages.MAX_EMAIL_LENGTH)
-    .matches(VALID_EMAIL_FORMAT).withMessage(ErrorMessages.EMAIL_INVALID_FORMAT)
+    .custom((value, { req }) => validateEmail(req.body["email"], 256))
 ];
 
 export const contact_email_validations = [
   body("contact_email").trim()
-    .not().isEmpty().withMessage(ErrorMessages.EMAIL)
-    .isLength({ max: 250 }).withMessage(ErrorMessages.MAX_EMAIL_LENGTH)
-    .matches(VALID_EMAIL_FORMAT).withMessage(ErrorMessages.EMAIL_INVALID_FORMAT)
+    .custom((value, { req }) => validateEmail(req.body["contact_email"], 250))
```

Comment:
Probably I will confirm with analyst why the difference between email length, I can not find the file related to the length of related chips columns

Reply:
That's a wider question about emails in the web app; this does not relate tot he work for this.

Reply:
Ok, agreed probably we should rise it to the standup